community/cypher/spec-suite-tools/src/test/scala/cypher/features/ScenarioTestHelper.scala

```diff
@@ -41,7 +41,7 @@ import scala.util.Success
 import scala.util.Try
 
 object ScenarioTestHelper {
-  var unexpectedSuccessCount = new AtomicInteger(0)
+  private val unexpectedSuccessCount = new AtomicInteger(0)
```

Comment:
The reference is not accessed outside of this object.

Interestingly the value of the atomic integer is not read so maybe this is here to support debugging.

Reply:
The CI builds failed, this is accessed by another testing tool later in the chain.